site/content/docs/0.0/helpers/icon.md

```diff
@@ -0,0 +1,396 @@
+---
+layout: docs
+title: Icon
+description: Quickly see the icon sizes you should use with a specific typography.
+group: helpers
+aliases:
+  - "/docs/helpers/icon/"
+toc: true
+---
+
+## Core concept
```

Comment:
I find this title quite different of what we usually have. 
Maybe "About icon sizes"?